packages/web-react/src/components/Grid/__tests__/Grid.test.tsx

```diff
@@ -49,4 +49,48 @@ describe('Grid', () => {
     const element = dom.container.querySelector('div') as HTMLElement;
     expect(element).toHaveClass('Grid--cols-2');
   });
+
+  it('should render with custom spacing', () => {
+    const dom = render(<Grid spacing="space-600" />);
+
+    const element = dom.container.querySelector('div') as HTMLElement;
```

Comment:
Can you please update tests as we do them now? Look into newest components.

We are using screen and https://testing-library.com/docs/queries/about#priority

Reply:
When I'm walking around… :) Tests are updated only for `Grid` component. I didn't want to increase this PR with the `GridItem` update tests as well.